public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java

```diff
@@ -262,7 +280,25 @@ public ConceptAnalysis apply(ConceptAnalysis ca) {
                 conceptService.searchConcepts(searchConceptsRequest.getQuery(), convertedConceptFilter,
                         searchConceptsRequest.getVocabularyIds(), domainIds, maxResults);
         ConceptListResponse response = new ConceptListResponse();
-        response.setItems(concepts.getContent().stream().map(TO_CLIENT_CONCEPT).collect(Collectors.toList()));
+        List<Concept> matchedConcepts = concepts.getContent();
+        for(Concept con : matchedConcepts){
```

Comment:
I don't think this logic is right. Fore every result that is not a standard concept , we want to get the standardConcepts and set it on the resp.standardConcepts? 
Don't we just want to do this if we matched on concept_code or concept_id and we only have one result ?

Reply:
Messed up a line. Added back.